BenchmarkCNV/BenchmarkCNV/BenchmarkCNV.wdl

```diff
@@ -0,0 +1,136 @@
+version 1.0
+
+workflow Benchmark_CNV_Caller {
+    input {
+        String truth_sample_name
+        String query_sample_name
+        File truth_vcf
+        File eval_vcf
+        File wittyer_config
+        File? bedfile
+        String wittyer_evaluation_mode
+        String wittyer_docker
+        String wittyer4mat_docker
+        Boolean run_wittyer4mat
+    }
+
+    # benchmark cnv.vcf and sv.vcf using witty.er tool
+    call BenchmarkCNV {
+        input:
+            wittyer_docker = wittyer_docker,
+            truth_vcf = truth_vcf,
+            truth_sample_name = truth_sample_name,
+            query_sample_name = query_sample_name,
+            eval_vcf = eval_vcf,
+            wittyer_config = wittyer_config,
+            wittyer_evaluation_mode = wittyer_evaluation_mode,
+            bedfile = bedfile
+    }
+
+    # wittyer4mat to parse the wittyer json output
+    if (run_wittyer4mat) {
+        call Wittyer4Mat {
+            input:
+                wittyer4mat_docker = wittyer4mat_docker,
+                wittyer_stats = BenchmarkCNV.wittyer_stats,
+                truth_sample_name = truth_sample_name
+        }
+    }
+
+    # Outputs that will be retained when execution is complete
+    output {
+        File wittyer_stats = BenchmarkCNV.wittyer_stats
+        File wittyer_annotated_vcf = BenchmarkCNV.wittyer_annotated_vcf
+        File wittyer_annotated_vcf_index = BenchmarkCNV.wittyer_annotated_vcf_index
+        Array[File]? Wittyer4Mat_event_stats = Wittyer4Mat.event_level_wittyer_stats
+    }
+    meta {
+        author: "Yueyao Gao"
+        email: "gaoyueya@broadinstitute.org"
+        description: "BenchmarkCNV.wdl is designed to evaluate the performance of Dragen CNV (Copy Number Variation) caller against GATK SV (Structural Variation) caller."
+    }
+}
+
+
+    # Task 1: Benchmark the large variant vcf against truth set
+    # If you are extracting vcf from a large callset vcf
+    # Checkout /BenchmarkCNV/SelectSampleFromCallSet.wdl
+    task BenchmarkCNV {
+
+        input {
+            String wittyer_docker
+            File truth_vcf
+            File eval_vcf
+            File wittyer_config
+            String wittyer_evaluation_mode
+            File? bedfile
+            String truth_sample_name
+            String query_sample_name
+            Int? mem
+            Int? disk_space
+            # If mem and disk size were not specified, use 4GB and 100 GB as default
+            Int mem_size = select_first([mem, 4])
+            Int disk_size = select_first([disk_space,100])
+    }
+        command <<<
+            set -e
+
+            if [[ -f "~{bedfile}" ]]; then
```

Comment:
ChatGPT suggests combining this if else in a way to reduce code duplication:

            # Build Wittyer command
            wittyer_cmd="/opt/Wittyer/Wittyer -i ~{evalVcf} \
            -t ~{truthVcf} \
            -em ~{wittyerEvaluationMode} \
            --configFile ~{wittyerConfig} \
            -o ~{truthSampleName}_wittyer_output"
            
            # Add --includeBed argument if bedfile is provided
            if [[ -f "~{bedfile}" ]]; then
                wittyer_cmd+=" --includeBed ~{bedfile}"
            fi
            
            # Execute Wittyer command
            eval $wittyer_cmd

Reply:
You can include `~{'--includeBed '+ bedfile} \` instead of the conditional in place of the first conditional block `--includeBed ~{bedfile} \` line